packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -61,6 +63,26 @@ function parseChromeLine(line: string): StackFrame | undefined {
   }
 }
 
+const CHROME_ANONYMOUS_FUNCTION_RE =
+  // eslint-disable-next-line max-len
+  /^\s*at ?((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\w+\.[cljs|cljc]|\/).*?)(?::(\d+))?(?::(\d+))??\s*$/i
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Couldn't we make a single regex for both cases with the function name and the url parentheses optional?

Reply:
I found making the `(` optional broke a bunch of the other edge cases that we tested for. I played around with some other ideas to no avail. 

Although not ideal, this was at least a working solution that didn't break any of the other cases. Happy to explore other ideas you might have?

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about extracting what we can have between paranthesis and generate a new regexp containing both options?

```js
const filePart =  "?:file|https?|blob|chrome-extension|native|eval|web..."
const regex = new Regex(`...(\(${filePart}\)|${filePart})...`)
```

Reply:
What @bcaudan suggests sounds great. I would do something like this:

```js
const url = '((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\\w+\\.|\\/).*?)'
const position = '((?::(\\d+))?(?::(\\d+))?'
const chromeLine = new Regex(`^\s*at(?: (.*?))? (?:\(${url}\)|${url})${position}\s*$`, 'i')
```